app/views/albums/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <div class="col-lg-12">
+    <table class="table table-hover">
+      <tbody>
+        <tr>
+          <th>Ranking</th>
+          <th>Name</th>
+          <th>Upvote</th>
+        </tr>
+        <% @albums.by_ranking.each do |album| %>
+          <tr>
+            <td>
+              Ranked: <%=album.ranking %>
+            </td>
+            <td>
+              <%= link_to "#{album.title}", album_path(album.id), action: :show %>
```

Comment:
I think for this we don't need `action: :show` part because the path information is entirely specified by the `album_path(album.id)` portion of this line.